codespell_lib/data/dictionary.txt

```diff
@@ -26530,6 +26570,7 @@ miminize->minimize
 miminized->minimized
 miminizes->minimizes
 miminizing->minimizing
+miminum->minimum
```

Comment:
Perhaps also `miminums->minimums` despite the formal plural is `minima`?

Reply:
I added both `miminums->minimums` and `mimina->minima`.